src/lib/functions/runtimes/js/builders/netlify-lambda.ts

```diff
@@ -1,16 +1,18 @@
-import { readFile } from 'fs/promises'
 import { resolve } from 'path'
 
 import { Command } from 'commander'
+import { type NormalizedPackageJson, readPackageUp } from 'read-package-up'
 
 import execa from '../../../../../utils/execa.js'
 import { fileExistsAsync } from '../../../../fs.js'
 import { memoizedBuild } from '../../../memoized-build.js'
+import type { BaseBuildResult } from '../../index.js'
 
-// @ts-expect-error TS(2525) FIXME: Initializer provides no value for this binding ele... Remove this comment to see the full error message
-export const detectNetlifyLambda = async function ({ packageJson } = {}) {
-  const { dependencies, devDependencies, scripts } = packageJson || {}
-  if (!((dependencies && dependencies['netlify-lambda']) || (devDependencies && devDependencies['netlify-lambda']))) {
+export type NetlifyLambdaBuildResult = BaseBuildResult
+
+export const detectNetlifyLambda = async ({ packageJson }: { packageJson: NormalizedPackageJson }) => {
```

Comment:
fun fact: I have zero clue what this whole file is about. what in the world is https://www.npmjs.com/package/netlify-lambda? is it super old and deprecated? did I waste my time? I should've looked into that earlier, but I'll get this merge for now and follow up.